app/views/cloud_volume/_common_new_edit.html.haml

```diff
@@ -112,6 +112,20 @@
 
 .form-group{"ng-if" => "vm.cloudVolumeModel.emstype == 'ManageIQ::Providers::Amazon::StorageManager::Ebs'"}
   %label.col-md-2.control-label
+    = _('Cloud Snapshot')
```

Comment:
Sorry, but I've missed this one before: please replace this with `Base Snapshot`.